[DOCS-8783] Updated Funnels and Pathways for Product Analytics (#27179)
DataDog/documentation: content/en/product_analytics/journeys/funnel_analysis.md

{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-filtering.png" alt="Use attributes to filter information when constructing your funnel" style="width:80%;" >}}

### Combining events

When constructing your funnel, you can combine multiple events within a given step, as end users may have different ways to achieve the same outcome through different flows.

To combine an event, click the three dots next to an event and select **+ Combine Events**.

## Refine conversion

You can further analyze the information on the funnel page to understand the conversion rate. Conversion rate is a crucial metric that measures the effectiveness of your site or application.

You can measure conversion by the following attributes:

- **Session count** or **users** - Understand how many sessions or users ended up completing the funnel.
- **Conversion count** - A count of users who went through the funnel you've defined.

> - **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of the total number of users that visited your site or application who converted.

Reviewer: ```suggestion
- **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of the total number of users that visited your site or application and converted.
```

Reviewer: just a suggestion.

  **Note**: Only completed RUM sessions contribute to conversion rates.
- **Drop-off rate** - A drop-off who left the page or app without finishing a process. This rate is the percentage of the total number of users who visited your site or application who did not complete the last step event.
- **Time to convert** - The time it took for the user to complete the step events.

You can measure these attributes **across all steps** or between **specific steps**.

{{< img src="product_analytics/journeys/funnel_analysis/funnels-across-steps.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}

Use the **filter** selector to filter by various criteria that you define.

> From there, you can click a datapoint to **investigate the specific attributes** that may have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.

Reviewer: ```suggestion
Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
```

## Changing the visualization

{{< img src="product_analytics/journeys/funnel_analysis/funnel-timeseries.mp4" alt="Click the visualization dropdown to select a different view" video=true >}}

> After you've defined the step events and conversion measurement, you can switch to see a different visualization and better understand conversion of users for your app.

Reviewer: ```suggestion
After you've defined the step events and conversion measurement, you can switch to a different visualization to better understand user conversions for your app.
```

Reviewer: minor suggestion. the "better understand conversion of users for your app" just sounded awkward when I read it out loud.

### Timeseries

> While understanding the overall conversion over a given period of time is helpful, you can use conversion over time to understand how the conversion has evolved over time.

Reviewer: ```suggestion
While understanding overall conversion during a specific period is helpful, analyzing conversion trends over time reveals how conversion has evolved.
```

> You can select the time period for which you want to graph the conversion and view it in percentages or in absolute count.

Reviewer: ```suggestion
You can select the time period for graphing the conversion and view conversions in percentages or in absolute count.
```

Reviewer: I assume "it" meant conversions and not time period. I might be wrong.
